projects/plugins/jetpack/_inc/client/recommendations/summary/index.jsx

```diff
@@ -82,7 +82,13 @@ const SummaryComponent = props => {
 				</h1>
 				{ summaryPrimarySections.map( ( { name, slugs } ) => (
 					<section key={ name } aria-labelledby={ `primary-onboarding-${ name }` }>
-						<h2 id={ `primary-onboarding-${ name }` }>Part of your { name } plan</h2>
+						<h2 id={ `primary-onboarding-${ name }` }>
```

Comment:
Not sure how much this matters, but this change probably belongs in the [base onboarding PR](https://github.com/Automattic/jetpack/pull/26484)